examples/with-tailwind/tailwind.config.js

```diff
@@ -0,0 +1,18 @@
+const colors = require("tailwindcss/colors");
+const path = require("path");
+
+module.exports = {
+  content: [
+    `${path.resolve(__dirname, "apps/")}/**/*.{js,ts,jsx,tsx}`,
```

Comment:
What happens here when we build just a single application? Does this run across all of them anyway?

Reply:
You're correct, this would be including styles from both apps. The new version will only look at the current app when building.